public/img/ic_pin.svg

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+  height="24"
+  width="24"
+  viewBox="0 0 24 24">
+  <path
```

Comment:
You're right. This is what it would look like at the same opacity:

![screenshot](https://tmp.dbrgn.ch/screenshots/20181010141552-03rnure9.png)

However, now it's lighter than the "back" arrow on the left and generally looks like it's disabled somehow. What do you think?

Reply:
It looks good to me.

Reply:
Done.

Reply:
Reverted it again, the "pinned" version looks bad with low contrast (it seems that the button is disabled).